### PR TITLE
docs: add code of conduct and contributing guide

### PR DESCRIPTION
**Pull request by ayuhito**
Adds `CODE_OF_CONDUCT.md` and `CONTRIBUTING.md` according to Hacktoberfest guidelines. Reorganises things a little in the process.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Overview
+
+The website uses [Ractive.js](https://ractive.js.org/), [ractive-route](https://github.com/MartinKolarik/ractive-route), and a custom build of [Bootstrap v3](https://getbootstrap.com/). To add a new page, you need to create a new `.html` in [views/pages](https://github.com/jsdelivr/www.jsdelivr.com/tree/master/src/views/pages) (use one of the existing ones as a reference), and add it to [client-side routing](https://github.com/jsdelivr/www.jsdelivr.com/blob/master/src/public/js/app.js). All `.html` files are compiled as [Ractive.js components](https://ractive.js.org/api/#component-files). Styles are in a separate [less directory](https://github.com/jsdelivr/www.jsdelivr.com/tree/master/src/public/less), which mirrors the `views` structure.
+
+### Guidelines
+
+ - Bug fixes and changes discussed in the existing issues are always welcome.
+ - For new ideas, please open an issue to discuss them before sending a PR.
+ - Make sure your PR passes `npm test` and has [appropriate commit messages](https://github.com/jsdelivr/www.jsdelivr.com/commits/master).
+ - A Heroku preview is built automatically for every PR and will be reviewed before merging.
```
Comment (jimaek) [inline CONTRIBUTING.md:14]: Maybe just a "A preview instance" without mentioning the provider

Comment (MartinKolarik) [reply CONTRIBUTING.md:14]: There's no instance at all for outside contributor PRs.

Comment (jimaek) [reply CONTRIBUTING.md:14]: render docs dont seem to mention this, they make it seem that any PR against master will create a preview

Comment (MartinKolarik) [reply CONTRIBUTING.md:14]: 🤷‍♂️

**Pull request merged**